DrawerKit/DrawerKit/Public API/Structs/DrawerState.swift

```diff
@@ -2,7 +2,7 @@ import UIKit
 
 /// An enumeration to describe the various states the drawer can be in.
 
-public enum DrawerState: Equatable { // the implementation of Equatable is internal
```

Comment:
I disagree with removing this comment. It was written for a reason, to hide implementation details from the public API.

Reply:
It became outdated by the removal of the custom implementation and IMO it does not provide any value even with it, also considering that it is `public` so the comment is confusing.
IMO in this case `Equatable` should not have been used and instead a custom comparison method should have been used if there is a need to account for tolerance, but I'll revert that change.